### ISSUE TITLE
A teeny-tiny change to loss.

### ISSUE THREAD

**Issue by justOleh** [TheodoreWolf/pinns#2]
I  think (but could wrong) there is a teeny-tiny term in loss function that I believe have no influence on gradient descent. 

The line https://github.com/TheodoreWolf/pinns/blob/main/src/cooling/network.py#L65 is 

`loss += self.loss2_weight + self.loss2_weight * self.loss2(self)`
I believe it should be 
`loss += self.loss2_weight * self.loss2(self)`

**Comment by TheodoreWolf** [comment]
You're correct, not sure how I missed this. I'll fix it. Funnily enough, adding a constant to the loss doesn't affect optimisation at all so this is probably why I never noticed this.

**Comment by TheodoreWolf** [comment]
Commit here 86c87020d43924ccf00ff836975a294a0757d6e6
